src/languages/pf.js

```diff
@@ -0,0 +1,58 @@
+/*
+Language: pf
+Category: config
+Author: Peter Piwowarski <oldlaptop654@aol.com>
+Description: The pf.conf(5) format as of OpenBSD 5.6
+*/
+
+function(hljs) {
+  var MACRO = {
+    className: 'variable',
+    begin: /\$[\w\d#@][\w\d_]*/
+  };
+  var TABLE = {
+    className: 'variable',
+    begin: /</, end: />/
+  };
+  var QUOTE_STRING = {
+    className: 'string',
+    begin: /"/, end: /"/
+  };
+
+  return {
+    aliases: ['pf', 'pf.conf'],
+    lexemes: /[a-z0-9_<>-]+/,
+    keywords: {
+      action: /* block match pass are "actions" in pf.conf(5), the rest are
```

Comment:
Last time we try to minimize different classes count across languages: https://github.com/isagalaev/highlight.js/issues/348#issuecomment-58228366. I suggest to merge `action` and `parameter` keywords into `built_in`.